blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticIO.java

```diff
@@ -150,20 +149,23 @@ private IndexRequestBuilder createSingleRequest(Discovery md) throws IOException
         
         List<SearchResult> results = new ArrayList<SearchResult>();
         Timer.Context searchTimerCtx = searchTimer.time();
-        
-        // todo: we'll want to change this once we decide and use a query syntax in the query string.
-        BoolQueryBuilder qb = boolQuery()
+        QueryBuilder qb;
+
+        GlobPattern pattern = new GlobPattern(query);
+        if (pattern.hasWildcard()) {
+            qb = wildcardQuery(metric_name.name(), query);
```

Comment:
So I was thinking a little differently here.

If no wild cards, do termQuery; otherwise do regex.

hasWildcard() returns true for any glob meta-character, doesn't it?  if so, ES wildcardQuery() doesn't support those does it?